Tests/fixtures/clienthints.yml

```diff
@@ -0,0 +1,79 @@
+---
+-
+  user_agent: Mozilla/5.0 (Linux; Android 8.1.0; ZC554KL) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/98.0.4758.87 Mobile Safari/537.36
+  headers:
+    Sec-CH-UA: '" Not A;Brand";v="99", "Chromium";v="98", "Google Chrome";v="98"'
+    Sec-CH-UA-Platform: "Android"
+    Sec-CH-UA-Mobile: "?1"
+    Sec-CH-UA-Full-Version: "98.0.4758.87"
+    Sec-CH-UA-Platform-Version: "8.1.0"
+    Sec-CH-UA-Model: "ZC554KL"
+    Sec-CH-Prefers-Color-Scheme: "light"
+  os:
+    name: Android
+    version: 8.1.0
+    platform: ""
+  client:
+    type: browser
+    name: Chrome Mobile
+    version: 98.0.4758.87
+    engine: Blink
+    engine_version: ""
+  device:
+    type: smartphone
+    brand: Asus
+    model: ZenFone 4 Max
+  os_family: Android
+  browser_family: Chrome
+-
+  user_agent: Mozilla/5.0 (Linux; arm_64; Android 8.1.0; ZC554KL) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/96.0.4664.174 YaApp_Android/22.14.1 YaSearchBrowser/22.14.1 BroPP/1.0 SA/3 Mobile Safari/537.36
+  headers:
+    Sec-CH-UA: '" Not A;Brand";v="99", "Chromium";v="96", "Yandex";v="22"'
+    Sec-CH-UA-Platform: "Android"
+    Sec-CH-UA-Mobile: "?1"
+    Sec-CH-UA-Full-Version: "22.1.4.706"
+    Sec-CH-UA-Platform-Version: "8.1.0"
+    Sec-CH-UA-Model: "ZC554KL"
+    Sec-CH-Prefers-Color-Scheme: "light"
+  os:
+    name: Android
+    version: 8.1.0
+    platform: ARM
+  client:
+    type: browser
+    name: Yandex Browser
+    version: 22.14.1
+    engine: Blink
+    engine_version: ""
+  device:
+    type: smartphone
+    brand: Asus
+    model: ZenFone 4 Max
+  os_family: Android
+  browser_family: Unknown
+-
+  user_agent: Mozilla/5.0 (Linux; Android 8.1.0; ZC554KL) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/96.0.4664.104 Mobile Safari/537.36 OPR/67.1.3508.63168
+  headers:
+    Sec-CH-UA: '" Not A;Brand";v="99", "Chromium";v="96", "Opera";v="82", "OperaMobile";v="67"'
+    Sec-CH-UA-Platform: "Android"
+    Sec-CH-UA-Mobile: "?1"
+    Sec-CH-UA-Full-Version: "96.0.4664.104"
```

Comment:
These are 3 tests that I did from my phone.
For some reason, Opera returns the Chrome version, not its own.

Reply:
I used this site for get information of CH
https://browserleaks.com/client-hints

Reply:
Tried Opera Mobile in my phone with Android 11. There Opera also returns the chrome version...

Reply:
Btw. another site to test the headers is https://user-agent-client-hints.glitch.me/